monad-eth-txpool-executor/src/forward.rs

```diff
@@ -71,6 +80,9 @@ impl Default for EthTxPoolForwardingManager {
 
             egress: VecDeque::default(),
             egress_waker: None,
+            egress_bytes_rate_limit: RateLimiter::direct(Quota::per_second(
+                EGRESS_RATE_LIMIT_BYTES_PER_SECOND,
```

Comment:
maybe rate limiter with burst would be a better fit, better to deliver txs on time before block proposal rather than slowly feed them